### DISCUSSION TITLE
How to display multilevel directory name for atomic.omp.json

### DISCUSSION THREAD

**Discussion by Jack012a** [JanDeDobbeleer/oh-my-posh#4699] [General]
Hello,

atomic.omp.json is my favorite theme for neovim.

I would like to display multilevel directory name for this theme which currently only shows one level.
Which part of the config I need to change to make this possible?

Thank you

**Comment by JanDeDobbeleer** [comment]
You need to adjust the path segment. Have a look at the [docs](https://ohmyposh.dev/docs/segments/path).
